internal/pkg/manifest/workload.go

```diff
@@ -289,3 +297,14 @@ func UnmarshalWorkload(in []byte) (interface{}, error) {
 		return nil, &ErrInvalidWorkloadType{Type: typeVal}
 	}
 }
+
+func buildRequired(image Image) (bool, error) {
+	hasBuild, hasURL := image.Build.isEmpty(), image.Location == nil
+	if hasBuild == hasURL {
+		return false, fmt.Errorf("either build or url in the manifest needs to be specified")
```

Comment:
Would it be overkill to either make it explicit that it has to be exactly one of these that is specified (not both), or even have a different error message if both are specified? A fairly unlikely scenario, but possible.

Reply:
could you give an example? i have trouble generating a concise version of this error message.

Reply:
```
if hasBuild && hasURL {
  return false, fmt.Errorf(`only one of either "image.build" or "image.location" can be specified in the manifest`)
}
if !hasBuild && !hasURL {
  return false, fmt.Errorf(`exactly one of either "image.build" or "image.location" must be specified in the manifest`)
```
Just a thought. This is probably overkill because "either" implies just one and it's highly unlikely a user would populate both fields. But in case we need to be very explicit we could do the above, or just use the wording of the second example for hasBuild == hasURL....

Reply:
yeah maybe a bit overkill...but thank you for bring that up!